packages/@lwc/engine-server/src/__tests__/fixtures/slot-forwarding/slots/mixed/modules/x/lightParent/lightParent.html

```diff
@@ -0,0 +1,6 @@
+<template lwc:render-mode="light">
+    <x-shadow-leaf>
+        <slot name="top" slot="bottom"></slot>
+        <slot name="bottom" slot="top"></slot>
```

Comment:
```suggestion
        <slot name="parent-top" slot="leaf-bottom"></slot>
        <slot name="parent-bottom" slot="leaf-top"></slot>
```
This would make it more clear which slot is which.

Reply:
I think the fact that we're using the same name here is kind of important. The old LWC engine used to get mixed up and "auto-forward" slots if, by chance, the parent had the same slot name as the child.